origin-js/src/models/listing.js

```diff
@@ -24,8 +24,8 @@ export class Listing {
    *  - {string} seller - address of the seller
    *  - {string} display - 'normal', 'featured', 'hidden'
    *  - {Array<Object>} media
-   *  - {Object} comission - consists of 'amount' and 'currency' properties
-   *  - {Array} slots
+   *  - {Object} commission - consists of 'amount' and 'currency' properties
```

Comment:
nit: (I forgot to do this) while you're here, could you add a comment for `commissionPerUnit`?

Reply:
sure 👍